README.md

```diff
@@ -5,102 +5,68 @@
 
 Speed up the creation and maintenance of your JS applications. Zero configuration way to use modern code-quality tools.
 
-+ **Zero-config.** Any tool already configured inside.
++ **Zero-config.** Any tool already configured for you.
 + **Universal.** Supports TS, React, and can be used with any tech.
-+ **Uniform.** Config can't be modified (all projects have the same configs).
++ **Uniform.** Config can be shared (all projects have the same configs).
 
 ## TL;DR
 
 ```sh
-yarn add -D @solid-soda/scripts@1
-yarn soda init
-
-yarn s lint
-yarn s pretty
+yarn dlx @solid-soda/scripts
 ```
 
-## Motivation
-
-1. Before start to code we must set-up ESLint, Stylelint, Prettier, Commitizen, etc. We can to automate all this. Machines have to suffer.
-2. New lint rule adding is a hell. We provide the one source of true for any project. Just bump `@solid-soda/scripts` in your project.
-
-## Install
+or if you prefer `npm`
 
 ```sh
-yarn add -D @solid-soda/scripts@1
-yarn soda init
+npx @solid-soda/scripts
 ```
 
-Or if you prefer `npm`:
-
-```sh
-npm install --save-dev @solid-soda/scripts@1
-npm run soda init
-```
+## Motivation
 
-## Scripts
+1. Before start to code we must set-up ESLint, Stylelint, Prettier, Commitizen, etc. We can to automate all this. Machines have to suffer.
+2. New lint rule adding is a hell. We provide the one source of true for any project. Just run `@solid-soda/scripts` more time in your project directory.
```

Comment:
We provide the one source of true for any project. Just run `@solid-soda/scripts` in your project directory.